CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.15.1 (July 2020)
+- Android 10 support
+- Update network libraries to more recent versions, OkHttp + Dav4Android
```

Comment:
We should use the current name: dav4jvm. Or using it in this way:

dav4jvm (old dav4Android)